src/Symfony/Component/Uid/Tests/UuidReturnTypeTest.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+namespace Symfony\Component\Uid\Tests;
+
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Uid\Uuid;
+
+/**
+ * @psalm-type NonEmptyString = non-empty-string
+ */
+class UuidReturnTypeTest extends TestCase
```

Comment:
I don't see much value in this test, tbh.

Reply:
@derrabus yes this return type does not make any sense here, I have removed it, thank you.

Reply:
No. The whole test ist pointless. Please remove it.

Reply:
@derrabus i have removed the test case, thank you